src/routes/DictionaryScreen.tsx

```diff
@@ -0,0 +1,88 @@
+import { StackNavigationProp } from '@react-navigation/stack'
+import React, { ReactElement, useState } from 'react'
+import { FlatList } from 'react-native'
+import { Subheading } from 'react-native-paper'
+import styled from 'styled-components/native'
+
+import { SadSmileyIcon } from '../../assets/images'
+import RouteWrapper from '../components/RouteWrapper'
+import SearchBar from '../components/SearchBar'
+import ServerResponseHandler from '../components/ServerResponseHandler'
+import Title from '../components/Title'
+import VocabularyListItem from '../components/VocabularyListItem'
+import { Document } from '../constants/endpoints'
+import labels from '../constants/labels.json'
+import useLoadAllDocuments from '../hooks/useLoadAllDocuments'
+import { RoutesParams } from '../navigation/NavigationTypes'
+
+const Root = styled.View`
+  padding: 0 ${props => props.theme.spacings.sm};
+`
+
+const ListEmptyContainer = styled.View`
+  align-items: center;
+  padding: ${props => props.theme.spacings.sm} 0;
+`
+
+const StyledSadSmileyIcon = styled(SadSmileyIcon)`
+  padding: ${props => props.theme.spacings.md} 0;
+`
+
+const Header = styled.View`
+  padding-bottom: ${props => props.theme.spacings.md};
+`
+
+interface Props {
+  navigation: StackNavigationProp<RoutesParams, 'DictionaryOverview'>
+}
+
+const DictionaryScreen = ({ navigation }: Props): ReactElement => {
+  const documents = useLoadAllDocuments()
+  const [searchString, setSearchString] = useState<string>('')
+  const filteredDocuments = documents.data?.filter(item => item.word.includes(searchString))
```

Comment:
I think we should do something similar to integreat and normalize the query and the word.

Reply:
Good point, thank you. Added it.